src/background.js

```diff
@@ -56,43 +56,52 @@ function downloadAgent() {
                         out: filename
                     };
                 }
-                await aria2.call("addUri", [downloadUrl], params).then(async ()=> {
-                    // Added successfully: Cancels and removes the download from chrome download manager
-                    chrome.downloads.erase({id: downloadItem.id});
+                await aria2.call("addUri", [downloadUrl], params).then(async () => {
+                    // Added successfully: Cancels and removes the download from browser download manager
+                    browser.downloads.erase({ id: downloadItem.id });
```

Comment:
Downloads.erase() does not seem to work with Firefox. ( Works on chrome/edge )

Reply:
@149segolte please add these as the comment in the code as well.